xapian-core/m4-macros/xapian-1.5.m4

```diff
@@ -31,7 +31,7 @@ AC_DEFUN([XO_LIB_XAPIAN],
   AC_ARG_VAR(XAPIAN_CONFIG, [Location of xapian-config (default:] ifelse([$3], [], xapian-config, [$3]) [on PATH)])
   dnl AC_PATH_PROG ignores an existing user setting of XAPIAN_CONFIG unless it
   dnl has a full path, so add special handling for such cases.
-  xapian_config_to_check_for="ifelse([$3], [], xapian-config, [$3])"
+  config_script_to_check_for="ifelse([$3], [], xapian-config, [$3])"
```

Comment:
I'm not sure why we're renaming a local variable here? Seems like an unnecessary change, given `XO_LIB_XAPIANLETOR` isn't the same as `XO_LIB_XAPIAN` (these lines in particular will always be different between the two, so it doesn't even make it easier to apply changes from -core to -letor).

Reply:
>I think this is only a local variable so we'll get away with this having the same name as in xapian.m4, but it would be less confusing to call this xapianletor_config_to_check_for or similar. Or rename both to something generic like config_script_to_check_for and keep the two sources more aligned.
>
@ojwb commented on one of the resolved conversations.